src/main/java/com/google/cloud/tools/maven/RunMojo.java

```diff
@@ -40,11 +43,18 @@
   /**
    * Path to a yaml file, or a directory containing yaml files, or a directory containing
    * WEB-INF/web.xml.
+   *
+   * @deprecated 1.2.2
```

Comment:
it'd be nice to explain here what the user needs to do to replace their app.yaml configs here.